layouts/authors/single.html

```diff
@@ -187,4 +197,21 @@ <h3 class="title-3 xs-mb-20">{{ .Site.Data.globals.enged_cta.title }}</h3>
     </div>
   </div>
 </section>
+
+<script type="text/javascript">
+  const toggleToolTip = (el) => {
+    if (el.classList.contains('tooltip')) {
```

Comment:
We can use the `toggle()` method here : https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/toggle

Reply:
Ah, I always forget about the `toggle()` method. Thanks!